R/check_merge_output.R

```diff
@@ -37,8 +39,20 @@ check_merge_output <- function(data, study, ...) {
       filter(STUDYID %in% study[i]) %>%
       group_by(!!!group_var) %>%
       mutate(row = row_number()) %>%
```

Comment:
Please switch `row_number()` to `dplyr::row_number()` to avoid `globalVariables` below.